src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,60 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService = (ManufacturerService) injector
+                .getInstance(ManufacturerService.class);
+        System.out.println("The result of the method getAll() "
+                + "from ManufacturerServiceImpl Class: ");
+        manufacturerService.getAll().stream().forEach(System.out::println);
+
+        Manufacturer getToyota = manufacturerService.get(6L);
```

Comment:
better to name variable just toyota with out get word.

Reply:
getToyota is bad naming for Manufacturer which represent toyota. Is better for method name not variable.
if you don't want to use just toyota lets rename getToyota  to recievedToyota